Makefile

```diff
@@ -113,21 +148,80 @@ check: style type test clean
 #
 # Instruction: Contributors will need to run...
 #
-# - "make installextras": if installing for the first time or want to
+# - "make installplus": if installing for the first time or want to
 #                         update to the latest dev-requirements or
 #                         other extra dependencies.
-# - "make install"      : if only installing the local source (after
+# - "make install.e"  : if only installing the local source (after
 #                         making some changes) to the source code.
 #--------------------------------------------------------------------
 
-install: clean
-	@echo "📀 Installing $(PACKAGE_NAME) from local source ... ⏳"
-	if [ -f $(REQ_FILE) ]; then python -m pip install -r $(REQ_FILE); fi
+# .PHONY: install.e
+install.e: clean
+	@echo "📀🟢🔵 Installing $(PACKAGE_NAME) from local source ... ⏳"
 	python -m pip install -Ue "."
 
-installextras: install
-	@echo "📀 Installing $(PACKAGE_NAME) extra-dependencies from PyPI ... ⏳"
-	if [ -f $(DEV_REQ_FILE) ]; then python -m pip install -r $(DEV_REQ_FILE); fi
+# .PHONY: install
```

Comment:
Is this stuff all needed? It feels like noise.

Reply:
The make file commands are like your API. Sometimes they will also require some restructuring. Some will be for most people to use and some only for a select few. 

If you have a contribution guideline, you could add the make commands, effectively smoothening out any steps that require extra attention. 

But installation as a make command is always very helpful.

Reply:
I would like you to remove these. I think probably the makefile was a mistake. It was meant to be small and easily readable. Now it has dozens of complex commands.